### ISSUE TITLE
When will publish a new package to NPM?

### ISSUE THREAD

**Issue by Fental** [flatiron/prompt#201]
I installed the newest package(version code is 1.0.0), but the following code was invalid.

```
//
  // Disable prompt's built-in SIGINT handling:
  //
  prompt.start({noHandleSIGINT: true});
  
  process.on('SIGINT', function() {
    console.log("This will execute when you hit CTRL+C");
    process.exit();
  });
```

**Comment by 3stacks** [comment]
I don't think this project is actively maintained. If you don't mind switching, enquirer (https://www.npmjs.com/package/enquirer) is a pretty nice alternative (last published 2 months ago too)

**Comment by Fental** [comment]
> I don't think this project is actively maintained. If you don't mind switching, enquirer (https://www.npmjs.com/package/enquirer) is a pretty nice alternative (last published 2 months ago too)

OK, let me have a try. Thanks.

**Comment by caub** [comment]
I don't understand how the `noHandleSIGINT` should work, and why it doesn't work anymore, maybe it's because the SIGINT handler should be passed to the readline instance, that is handled through the read package:
https://github.com/npm/read/blob/main/lib/read.js#L64

@rubbingalcoholic If you've a bit of time, that would be cool if you can help us figure out how this option should work? I think you implemented it in prompt
